docs/users.md

```diff
@@ -16,10 +16,11 @@ specific dataset.
 * [Ocopea Cloud Native Developer Toolkit](https://ocopea.github.io/)
 * [Ocopea Architecture](architecture.md)
 
-# Install Guide
+# Product Documentation
 
-* [Installation guide TBD](tbd.md)
-* [How to Use TBD](tbd.md)
+* [Ocopea Kubernetes Extension](https://github.com/ocopea/kubernetes)
+* [Ocopea Cloud Foundry Extension](https://github.com/ocopea/cloudfoundry)
```

Comment:
Probably should say  Install Ocopea on Kubernetes and Install Ocopea on Cloud Foundry to clarify where to find install/deploy instructions.